cmd/clusterawsadm/cmd/eks/addons/list-available.go

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Can we rename this file (and the other similar) to replace the `-` to either `_` or using camal case to match the rest of the files in the repo?

Reply:
I've renamed to use `_`.